Mail2Bug/MessageProcessingStrategies/SimpleBugStrategy.cs

```diff
@@ -187,6 +197,22 @@ private static TempFileCollection SaveAttachments(IIncomingEmailMessage message)
             return attachmentFiles;
         }
 
+        private IWorkItemManager InitWorkItemManager() {
+            IWorkItemManager workItemManager;
+            if (_config.TfsServerConfig.SimulationMode)
```

Comment:
Any specific reason we need to duplicate this code here? We should rather keep this in Mail2BugEngine itself so that these classes are agnostic of which IWorkItemManager is being used.